### PR TITLE
feat: cart

### PR DESCRIPTION
**Pull request by Muzammil-khan710**
Cart page

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -48,4 +48,23 @@
 .overlay-text {
     position: absolute;
     top: 3rem;
-}
\ No newline at end of file
+}
+
+
+.cart-img {
+    height: 200px;
```
Comment (palash23jain) [inline styles.css]: rem/em can be preferred. But not a breaking change

**Review by palash23jain — APPROVED** [review]
code looks good to go

File: pages/cart.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+          </div>
+        </div>
+
+        <div class="input-container">
+            <h1>Your Cart</h1>
+
+            <div class="cart-container">  
+            <div class="card">
+                <div class="horizontal-card cart-card">
+                <img class="cart-img" src="https://images.pexels.com/photos/996329/pexels-photo-996329.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500" alt="">
```
Comment (dhimanaishwarya) [inline pages/cart.html]: you can add alt for images

**Review by dhimanaishwarya — APPROVED** [review]
Looks good to merge

**Pull request merged**